docs/source/providers.rst

```diff
@@ -215,6 +215,17 @@ The ``Registry`` interface represents registry information. It has the following
 * ``asset_id`` - ``string``: The ID of the asset in the registry.
 * ``asset_info`` - ``AssetInfo``: Information about the asset in the registry.
 
+**Filter**
+
+The ``Filter`` interface represents a filter with a unique identifier. It has the following properties:
+
+* ``id`` - ``string``: The unique identifier for the filter.
+
+**FilterDeletionStatus**
+
+The ``FilterDeletionStatus`` interface represents the status of a deletion operation. It has the following properties:
```

Comment:
The interface name is FilterDeletionResult in the code.

Reply:
e26dfd2a3a9d699b993a4755f51bbf6970d32e80